qchem/pennylane_qchem/qchem.py

```diff
@@ -414,7 +414,8 @@ def _terms_to_qubit_operator(coeffs, ops):
     for coeff, op in zip(coeffs, ops):
 
         # wire ids
-        wires = op.wires.tolist()
+        wires = op.wires.tolist()  # Can we use subsystems here? Otherwise the Qchem library relies on users
+                                   # and devices assuming consecutive indices as wires
```

Comment:
Would it be best to use wire objects/labels throughout the QChem library?

Reply:
🤔

Reply:
I agree. On my radar.